attributes/default.rb

```diff
@@ -1,8 +1,8 @@
 #
-# Cookbook Name:: nodejs
+# Cookbook:: nodejs
 # Attributes:: nodejs
 #
-# Copyright 2010-2012, Promet Solutions
+# Copyright:: 2010-2017, Promet Solutions
```

Comment:
Not sure if this makes sense. Not sure if Marius even still works for this company, and what parts of changes would be copyrighted by this company.